app/assets/stylesheets/active_admin_sidebar_pure_icons.scss

```diff
@@ -0,0 +1,62 @@
+/* .icono-caretLeft */
+.icono-caretRight,
+.icono-caretLeft {
+  width: 19px;
+  height: 19px;
+}
+.icono-caretRight:before,
+.icono-caretRight:after,
+.icono-caretLeft:before,
+.icono-caretLeft:after {
+  width: 8px;
+  height: 2px;
+  position: absolute;
+  bottom: 1px;
+  margin: auto 0;
+  right: 6px;
+  box-shadow: inset 0 0 0 32px;
+  transform-origin: right;
+}
+.icono-caretRight:before,
```

Comment:
Selector `icono-caretRight` should be written in lowercase with hyphens<br>Selector `icono-caretLeft` should be written in lowercase with hyphens